src/main/kotlin/ink/abb/pogo/scraper/tasks/ReleasePokemon.kt

```diff
@@ -23,19 +23,20 @@ class ReleasePokemon : Task {
         val groupedPokemon = ctx.api.inventories.pokebank.pokemons.groupBy { it.pokemonId }
         val ignoredPokemon = settings.ignoredPokemon
         val obligatoryTransfer = settings.obligatoryTransfer
+        val forceTransfer = settings.shouldForceTransfer
         val minIVPercentage = settings.transferIVThreshold
         val minCP = settings.transferCPThreshold
 
         groupedPokemon.forEach {
-            val sorted = it.value.sortedByDescending { it.cp }
+            val sorted = if (forceTransfer) it.value else it.value.sortedByDescending { it.cp }
```

Comment:
This can be removed, because if `forceTransfer=true` it doesn't matter if the pokemons are sorted.